docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
           - teachcomputing.test
           - teachcomputing.rpfdev.com
   db:
-    image: postgres:12.9
+    image: postgres:15
```

Comment:
Could you please specify the minor version here.

I've had an experience in a project which didn't have the image specified particularly tightly which resulted in me using an older version of postgres than expected because it had been pulled for a different project at a previous time.